deploy/olm-catalog/ibm-management-ingress-operator/1.1.0/ibm-management-ingress-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -39,6 +39,29 @@ metadata:
             },
             "routeHost": ""
           }
+        },
+        {
+          "apiVersion": "operator.ibm.com/v1alpha1",
+          "kind": "OperandRequest",
+          "metadata": {
+              "finalizers": [
```

Comment:
@hmtai We should remove the `fnalizers` section. It will be added by ODLM when `OperandRequest` is being removed.

Reply:
Yes,remove it.